### PR TITLE
[DEV-1757] Only output one line for success of worker plugins

### PR DESCRIPTION
**Pull request by jsignell**
This PR collapses the output from the `SaturnSetup` plugin so that it's less overwhelming.

image

### CODE REVIEW

**Comment by hhuuggoo** [comment]
Task linked: [DEV-1757 Dask worker plugin output is a bit overwhelming and confusing for large clusters](https://app.clickup.com/t/1250339/DEV-1757)

File: dask_saturn/core.py [hunk 1]
```diff
@@ -312,9 +312,19 @@ def _start(
     def register_default_plugin(self):
         """Register the default SaturnSetup plugin to all workers."""
         log.info("Registering default plugins")
+        outputs = {}
         with Client(self) as client:
             output = client.register_worker_plugin(SaturnSetup())
-            log.info(output)
+            outputs.update(output)
+        output_statuses = [v["status"] for v in outputs.values()]
+        if "OK" in output_statuses:
```
Comment (jameslamb) [inline dask_saturn/core.py:320]: I don't understand this so I'm not sure how to review it, sorry. Could you show me an example of what an `output`  dict looks like? I didn't understand if the image in the PR description showed the expected logs **as of this PR** or the logs **without this PR** that you'd like to make shorter.

**Review by jameslamb — COMMENTED** [review]
I just tried this on internal.

The first time it ran, the workers had not started up yet: 
image

After waiting for the workers to come up, this printed the "Success" message

image

I left a question though. I'm not really sure what success looks like, so I can't approve this yet. Once you answer that question I'll be happy to come back and approve based on that.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 repos:
-  - repo: https://github.com/ambv/black
-    rev: stable
```
Comment (jsignell) [inline .pre-commit-config.yaml:3]: This seems to be the recommended approach based on: https://pre-commit.com/#using-the-latest-version-for-a-repository

File: dask_saturn/core.py [hunk 3]
```diff
@@ -280,9 +280,7 @@ def _start(
         logged_warnings: Dict[str, bool] = {}
         while self.cluster_url is None:
             response = requests.post(
-                url + url_query,
-                data=json.dumps(cluster_config),
-                headers=self.settings.headers,
+                url + url_query, data=json.dumps(cluster_config), headers=self.settings.headers,
```
[the remark this answers is not in the document]
Comment (jameslamb) [reply dask_saturn/core.py]: sorry, I see you updated precommit while I was reviewing haha

[the remark this answers is not in the document]
Comment (jsignell) [reply dask_saturn/core.py]: It was pre-commit being annoying. I included the update that it wanted in this PR now.

File: dask_saturn/core.py (hunk 1 above)
Comment (jsignell) [reply dask_saturn/core.py:320]: Does the context of the linked ticket not provide this?

Comment (jameslamb) [reply dask_saturn/core.py:320]: oh! totally missed that, sorry. I'm not used to there being tickets linked for these public repos. That describes it perfectly, thank you.

**Review by jameslamb — APPROVED** [review]

File: dask_saturn/core.py (hunk 1 above)
Comment (jsignell) [reply dask_saturn/core.py:320]: No worries, the output is like: `{'tcp://10.0.0.84:33237': {'status': 'OK'}}`

**Pull request merged**